index.html

```diff
@@ -198,158 +161,238 @@ <h2>Tonic</h2>
+            </li>
+            <li>
+              <a href="#"
+                ><img class="v-social-media" src="images/vec.png" alt="vector"
+              /></a>
             </li>
           </ul>
-        </form>
+          <button type="button" class="click-btn">Get my resume</button>
+        </section>
+        <section>
+          <ul class="lang">
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Languages</p>
+                <img src="images/arrows-down.png" alt="arrow_bottom" />
+              </div>
+              <ul class="languages">
+                <li>
+                  <img src="images/js.png" alt="JavaScript" />
+                  <a href="#">JavaScript</a>
+                </li>
+                <li>
+                  <img src="images/html..png" alt="CSS" />
+                  <a href="#">HTML</a>
+                </li>
+                <li>
+                  <img src="images/css..png" alt="CSS" />
+                  <a href="#">CSS</a>
+                </li>
+              </ul>
+            </li>
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Frameworks</p>
+                <img src="images/arrow.png" alt="arrow_bottom" />
+              </div>
+              <hr />
+            </li>
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Skills</p>
+                <img src="images/arrow.png" alt="arrow_bottom" />
+              </div>
+              <hr />
+            </li>
+          </ul>
+        </section>
       </div>
+    </section>
+    <footer>
+      <section class="contact">
+        <div class="contact-content">
+          <h1 class="contact-me">Contact me</h1>
+          <p class="paragraph item">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form
+          class="form-section"
+          action="https://formspree.io/f/mqknjqvv"
+          method="post"
+        >
+          <input
+            id="name"
+            class="text-holder1"
+            type="text"
+            name="name"
+            maxlength="30"
+            required
+            placeholder="Name"
+          />
+          <input
+            id="email"
+            class="text-holder1"
+            type="email"
+            name="name"
+            required
+            placeholder="Email"
+          />
+          <textarea
+            id="message"
+            class="text-area"
+            placeholder="Write your message here"
+            maxlength="500"
+            required
+          ></textarea>
+          <div id="form-button">
+            <button
+              id="clicked-btn"
+              class="click-btn"
+              type="submit"
+              name="sumit"
+            >
```

Comment:
- [ ] You applied the changes requested, however, your button is not aligned to the center. Please, fix it to stick to the Figma design.

**Your button**
![Screenshot from 2022-07-16 18-42-10](https://user-images.githubusercontent.com/80612925/179364434-c0fe81b4-553b-4252-919b-82178cc92a85.png)
**The Figma**
![image](https://user-images.githubusercontent.com/80612925/179364441-2f70b745-1e9d-4307-8480-8890567aead5.png)

Reply:
Hello @Hirwacardinalloic I can see that the button still is not positioned at the center. Adding the code below in the media query will solve this.

```
#clicked-btn {
    align-items: center;
    width: 142px;
    height: 48px;
    margin: 2rem auto 0 9rem;
  }
```

Reply:
I have tried to reach out to you via slack and you seem not to be responsive.